src/components/animated/_spotlight.less

```diff
@@ -0,0 +1,151 @@
+@import "./_variables.less";
+@colors : @blue,@pink,@red,@green,@yellow,@orange,@purple,@black;
+@colorbtns:blue-btn,pink-btn,red-btn,green-btn,yellow-btn,orange-btn,purple-btn,black-btn;
+.spotlight-colorspecific-mixin(@param) when (@param = "main")
+{
+	each(@colorbtns,{
+		&.@{value}{
+			@currentcolor:extract(@colors,@key);
+			border-color:@currentcolor;
+			box-shadow:inset 0px 0px 10px @currentcolor;	
+		}
+	})
+} 
+.spotlight-colorspecific-mixin(@param) when (@param = "after/before")
+{
+	each(@colorbtns,{
+		&.@{value}{
+			@currentcolor:extract(@colors,@key);
+			&:after{
+			border-color:@currentcolor;
+			box-shadow:inset 0px 0px 10px @currentcolor;
+			}
+			&:before{
+				background-color: extract(@colors,@key);
+				border-color:white ;
+			}	
+		}
+	})
+} 
+.sbtn.spotlight-btn{
+	position: relative;
+	padding:10px;
+	color:black;
+	height:unset;
+	width:unset;
+	background-color: transparent;
+	overflow:visible !important;
+	cursor:pointer;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	outline:none;
+	border-width: 3px; 
+	border-style: solid;
+	z-index: 0;
```

Comment:
Remove lines 32, 34, 35, 38, 39, 40, 41, 42, 43, 44, 45 as they are unnecessary.

Also, change `color: black` to `color: @darkText`

Reply:
The stack effect won't work if line 45 is not included

Reply:
ok you can leave that line

Reply:
@mrSidSat you didn't remove line 32 about the padding

Reply:
it is removed in the latest commit